web_source/cities/africa/ethiopia/axum.md

```diff
@@ -4,12 +4,15 @@ Country: Ethiopia (Africa)
 
 ##### Also known as:
 
-FIXME
+The Aksumite Empire
 
 ### What we would like to see there
 
-FIXME
+- Northern Stelae Park
+- Chapel of the Tablet
```

Comment:
would be nice to show an image, such as  ![](https://img.atlasobscura.com/YFgcLxLsipbbuzCZnL5blC5n3SYJVUBLaa-o6RdLG6Q/rt:fit/h:390/q:81/sm:1/scp:1/ar:1/aHR0cHM6Ly9hdGxh/cy1kZXYuczMuYW1h/em9uYXdzLmNvbS91/cGxvYWRzL3BsYWNl/X2ltYWdlcy8wY2Y3/ZDk3NDNkNjkzNGRh/ZjRfMjg0MjYxMDgw/M183ZjdhYmZiNjVk/X2IuanBn.jpg)